.clusterfuzzlite/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# c-toxcore Clusterfuzzlite build environment
+
+# We want to use the latest tools always
+# hadolint ignore=DL3006
```

Comment:
Don't disable this. Use ":latest" below instead.

Reply:
Should be fixed now